app/src/RemoteConfig.ts

```
@@ -0,0 +1,209 @@
+    console.log('Remote config fetch failed', err);
+  }
+};
+
+export const getFeatureFlag = async <T extends FeatureFlagValue>(
+  flag: string,
+  defaultValue: T,
+): Promise<T> => {
+  try {
+    // Check local overrides first
+    const localOverrides = await getLocalOverrides();
+    if (localOverrides.hasOwnProperty(flag)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)